src/cogs/owner.py

```diff
@@ -321,10 +334,17 @@ async def loadbab(self, ctx: Context):
                                     default_palette.shape[1])
                 color_x, color_y = int(color_x), int(color_y)
             
-            if len(tile['sprite']) > 1:
-                print(f"Bab tile {name} has more than one sprite")
-            
-            sprite_name = name.replace("/", "_").replace("'", "_").replace(" ", "_")
+            sprite_name = (
+                name.replace("<", "lt")
+                    .replace(">", "gt")
+                    .replace(":", "colon")
+                    .replace("\"", "quot")
+                    .replace("/", "sol")
+                    .replace("\\", "bsol")
+                    .replace("|", "vert")
+                    .replace("?", "quest")
+                    .replace("*", "ast")
+            )
```

Comment:
This seems like it could become a problem (also, no handling of `'`?)

Reply:
what do you think i should do?

Reply:
I'm not sure, it's just that this doesn't catch all special characters and endlessly replace chaining seems like a bad idea 
I'd prefer something that just applies to certain character ranges but I'm not sure where the character names would come from

Reply:
Resolved by latest commit